tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -1345,7 +1345,8 @@ def test_consume(self):
                                    timeout=self.msg_timeout)
 
             self.stage_lots_of_failed_consumers()
-            self.stage_hard_restart(producer)
+            RedpandaServiceCloud.concurrent_restart_pods(producer)
+            RedpandaServiceCloud.verify_basic_produce_consume(producer)
```

Comment:
My guess is this might work if there is a static id present in `globals.json`.
I suggest we update it:
```
self.redpanda.verify_basic_produce_consume(producer)
```

Reply:
Yes, that was supposed to be self.repdanda
Updated and tested